### ISSUE TITLE
Invalid Route

### ISSUE THREAD

**Issue by bryantwells** [aelvan/mailchimp-subscribe-craft#58]
Submitting the following form results in a 404 error:

```
<form class="ContactForm" action="" method="POST">
    {{ csrfInput() }}
    <input type="hidden" name="action" value="mailchimp-subscribe/list/Subscribe"> 
    <!-- ... -->
    <input type="submit" name="" value="Subscribe"/>
</form>
```

```
HTTP 404 – Not Found – yii\web\NotFoundHttpException
Page not found.
Caused by: Invalid Route – yii\base\InvalidRouteException
Unable to resolve the request "mailchimp-subscribe/list/Subscribe".
```

I am assuming the value in the the hidden input is incorrect — but can't seem to find the correct value. Any help would be appreciated!

**Comment by bryantwells** [comment]
This was an error on my end. I had a capital 'S' in the subscribe value.

**Comment by bryantwells** [comment]
Reopening after realizing that hidden input in the second code block under the [Example Usage](https://github.com/aelvan/mailchimp-subscribe-craft#example-usage) heading within the readme has an incorrect value.

`value="mailchimp-subscribe/list/Subscribe"` should be `value="mailchimp-subscribe/list/subscribe"`

**Comment by aelvan** [comment]
Fixed in 2.0.2.
